### ISSUE TITLE
Checkbox enabled

### ISSUE THREAD

**Issue by FlowinBeatz** [iMi-digital/imi_newsletter_form#1]
Danke für die tolle Extension!
Es wäre mega gut, wenn man die "Newsletter abonnieren" Funktion noch per Checkbox aktivieren könnte.

Vielleicht nimmt sich ja jemand der Sache an :-) # # #

**Comment by amenk** [comment]
Gute Idee. Vielleicht willst du einen Pull Request machen :) ?

**Comment by FlowinBeatz** [comment]
Ich habe leider 0,0 Backend Skills und das ganze grob zusammengefrickelt aus zwei Formularen und JS...

**Comment by GMTeams** [comment]
Auch ich bedanke mich für diese nützliche Extension.

Ich bräuchte ebenfalls eine Checkbox, implementiert wurde das noch nicht, oder?
Ansonsten wäre der workaround von @030media für mich interessant.

**Comment by FlowinBeatz** [comment]
Ich habe es tatsächlich aus zwei Formularen und JS zusammengebaut.
Du kannst es Dir gerne hier anschauen und versuchen nachzubauen: https://www.steinfels.de/katalog.html (unten beim PDF Katalog)

**Comment by GMTeams** [comment]
Ah okay im groben denke ich verstehe ich schon die Herangehensweise.
Vielen Dank für die rasche Antwort!

**Comment by FlowinBeatz** [comment]
Einfach zwei Formulare einbauen und bei klick auf die Checkbox die Sichtbarkeit togglen, that's it.
Dabei dann auch beachten, dass die values beider Input Felder immer gespiegelt werden per JS, damit jemand der mittendrin die Checkbox klickt nicht seine Eingaben verliert.
Eine Serverbasierte Lösung wäre hier natürlich schlauer.

**Comment by GMTeams** [comment]
Klar, serverbasiert wär's sicher schöner, aber so geht's notfalls auch. Danke nochmal.

**Comment by amenk** [comment]
Nutzt die Extension noch jemand :) ?
